controllers/argocd/notifications/configmap.go

```diff
@@ -0,0 +1,81 @@
+package notifications
+
+import (
+	"github.com/argoproj-labs/argocd-operator/common"
+	"github.com/argoproj-labs/argocd-operator/pkg/argoutil"
+	"github.com/argoproj-labs/argocd-operator/pkg/cluster"
+	"github.com/argoproj-labs/argocd-operator/pkg/workloads"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+func (nr *NotificationsReconciler) reconcileConfigMap() error {
+
+	nr.Logger.Info("reconciling configMaps")
+
+	name := argoutil.GenerateUniqueResourceName(nr.Instance.Name, nr.Instance.Namespace, ArgoCDNotificationsControllerComponent)
+	labels := common.DefaultLabels(name, nr.Instance.Name, ArgoCDNotificationsControllerComponent)
```

Comment:
@ciiay please move these to the main `notifications_controller.go` file before the reconciliations are triggered 
create global variables called `resourceName` and `resourceLabels` which can be accessed throughout the package